example/DSSAT/original/PETPT_DSSAT.f

```diff
@@ -0,0 +1,315 @@
+C=======================================================================
```

Comment:
This is exactly the code you have to give in Fortran.

Reply:
I thought that it just the algorithm we will put in the file. Ok I will do it